frontend_tests/casper_tests/11-admin.js

```diff
@@ -130,6 +130,54 @@ casper.waitWhileSelector('.emoji_row', function () {
     casper.test.assertDoesntExist('.emoji_row');
 });
 
+
+function get_suggestions(str) {
+    casper.then(function () {
+        casper.evaluate(function (str) {
+            $('.create_default_stream')
+            .focus()
+            .val(str)
+            .trigger($.Event('keyup', { which: 0 }));
+        }, str);
+    });
+}
+
+function select_from_suggestions(item) {
+    casper.then(function () {
+       casper.evaluate(function (item) {
+            var tah = $('.create_default_stream').data().typeahead;
+            tah.mouseenter({
+                currentTarget: $('.typeahead:visible li:contains("'+item+'")')[0]
+            });
+            tah.select();
+        }, {item: item});
+    });
+}
+
+//Test default stream creation and addition
+
+casper.then(function () {
+    casper.click('#settings-dropdown');
+    casper.click('a[href^="#subscriptions"]');
+    casper.click('#settings-dropdown');
+    casper.click('a[href^="#administration"]');
+    var stream_name = "Scotland";
+    // It matches with all the stream names which has 'O' as a substring (Rome, Scotland, Verona etc).
+    // I used 'O' to make sure that it works even if there are multiple suggestions.
+    // Capital 'O' is used instead of small 'o' to make sure that the suggestions are not case sensitive.
+    get_suggestions("O");
+    select_from_suggestions(stream_name);
+    casper.waitForSelector('.default_stream_row', function () {
```

Comment:
can you refine these selectors to only be waiting for a `.default_stream_row` that has name `Scotland`?  That would make things a lot cleaner if we later change the default text fixtures to contain at least one existing default stream.